lib/queries/utils/buildQuery.js

```diff
@@ -8,6 +8,45 @@ import {
   REFERENCES_VALUES_MAP,
 } from '../../constants';
 
+export const getQueryStrings = (indexData, queryTemplate, filters, seeAlsoJoin = 'or') => {
```

Comment:
Let's rename it to `formatHeadingsQuery`, because this function only handles fields like `personalNameTitle`, `corporateNameTitle` etc